qp_qiime2/__init__.py

```diff
@@ -21,176 +25,171 @@
 
 # Define the rarefy command
 req_params = {
-    'i-table': ('artifact', ['BIOM']),
-    'p-sampling-depth': ['integer', 1000]
+    'BIOM table': ('artifact', ['BIOM']),
+    'Sampling depth': ['integer', 1000]
```

Comment:
is 1000 the default? it looks like this is a replicated value with the `dflt_param_set`? why 1000?